docker/README.md

```diff
@@ -5,15 +5,17 @@ for testing & development purposes.
 
 ### Building 
 
-First, you need to build `teleport:latest` Docker image. This image is built
-when you type `make build` BUT...
+First, you need to build `teleport:latest` Docker image.
```

Comment:
Do we need `teleport:latest` or `FROM teleport-buildbox:go1.13.2 `

Reply:
I think it is `teleport:latest`?:

![Screenshot from 2020-03-03 14-05-59](https://user-images.githubusercontent.com/43280172/75824417-543cd700-5d58-11ea-9d4b-0fd913b06616.png)

Reply:
Yes, it is. It's a local tag rather than anything pushed elsewhere so it doesn't matter too much.